userapi/userapi_test.go

```diff
@@ -468,15 +466,6 @@ func TestDevices(t *testing.T) {
 			name:      "explicit local user",
 			inputData: &api.PerformDeviceCreationRequest{Localpart: "test2", ServerName: "test", AccessToken: util.RandomString(8), NoDeviceListUpdate: true},
 		},
-		{
-			name:      "dupe token - ok",
-			inputData: &api.PerformDeviceCreationRequest{Localpart: "test3", ServerName: "test", AccessToken: dupeAccessToken, NoDeviceListUpdate: true},
-		},
-		{
-			name:      "dupe token - not ok",
-			inputData: &api.PerformDeviceCreationRequest{Localpart: "test3", ServerName: "test", AccessToken: dupeAccessToken, NoDeviceListUpdate: true},
-			wantErr:   true,
-		},
```

Comment:
These tests are not relevant to msc3861 as access tokens are not involved in this flow